flexmeasures/api/v3_0/tests/test_sensor_schedules.py

```diff
@@ -347,6 +347,7 @@ def test_get_schedule_fallback(
     Test if the fallback job is created after a failing StorageScheduler call. This test
     is based on flexmeasures/data/models/planning/tests/test_solver.py
     """
+    app.config["FLEXMEASURES_FALLBACK_REDIRECT"] = True
```

Comment:
This setting behaviour (together with resetting at the end of the test) assumes the config setting was originally set to `False`. We should check that explicitly.